src/odemis/gui/win/acquisition.py

```diff
@@ -695,8 +700,13 @@ def __init__(self, parent, orig_tab_data):
         # Set parameters for tiled acq
         self.overlap = 0.2
         try:
-            stage_rng = self._main_data_model.stage.getMetadata()[model.MD_POS_ACTIVE_RANGE]
-            # left, top, right, bottom
+            stage_md = self._main_data_model.stage.getMetadata()
+            stage_rng = stage_md[model.MD_POS_ACTIVE_RANGE].copy()
+            # Hack: Limit active range with overview map range if it's in metadata, until it's provided in the GUI
+            if model.MD_OVERVIEW_RANGE in stage_md and {'x', 'y'}.issubset(stage_md[model.MD_OVERVIEW_RANGE].keys()):
+                stage_rng.update(stage_md[model.MD_OVERVIEW_RANGE])
+
+            # left, top, right, bottom expected
```

Comment:
It's more like "left, bottom, right, top" (and that's the right order to store them).